tests/serverpod_test_server/lib/src/models/defaults/exception/default_exception.spy.yaml

```diff
@@ -0,0 +1,5 @@
+exception: DefaultException
+fields:
+    defaultMessage: String, default='Default error message'
```

Comment:
Suggestion: Add all supported data types to the exception.

This asserts we always support everything even if it is removed from models.

Reply:
But i would say it is enought with just the `default` keyword for all the different types.